.github/workflows/end_to_end_tests.yml

```diff
@@ -3,10 +3,10 @@ name: "Kubernetes End to End Tests"
 on:
   push:
     branches:
-      - master
+      - arthur/new-checks
```

Comment:
I assume you will change this back to master before / after merging this branch?